packages/webcomponents/src/components/payments-list/payments-list-filters.tsx

```diff
@@ -10,13 +11,21 @@ export class PaymentsListFilters {
   @Prop() setParamsOnChange: (name: string, value: string) => void;
   @Prop() clearParams: () => void;
 
+  @State() createdAfterInputValue: string;
+  @State() createdBeforeInputValue: string;
+
   private debouncedSetParamsOnChange: (name: string, value: string) => void;
 
   componentWillLoad() {
     // debounced input handler for text input
     this.debouncedSetParamsOnChange = debounce(this.setParamsOnChange, 300);
   }
 
+  handleDateInput = (name: string, value: string) => {
```

Comment:
It looks like it's working in both payments and payouts list, but in `payouts-list-filters` this function is setting the `createdAfterInputValue` and `createdBeforeInputValue` state. 
Should we also set this in here?

Reply:
Thanks @TiagoFuelber - after this comment I realized we don't actually need these state variables at all in this file. So its a bit simpler now